source/features/embed-gist-via-iframe.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	const embedMenuItem = select('.select-menu-item')!;
+	select('.select-menu-button')!.textContent = 'Embed via <script>';
+	select('.select-menu-item-heading', embedMenuItem)!.innerHTML = 'Embed via <code>&lt;script&gt;</code>';
+	select('.description', embedMenuItem)!.innerHTML = 'Embed this gist in your website via <code>&lt;script&gt;</code>.';
+
+	const iframe = `<iframe src="https://gist.github.com${location.pathname}.pibb"></iframe>`;
+	embedMenuItem.insertAdjacentElement(
+		'afterend',
+		<button name="button" type="button" className="select-menu-item width-full" aria-checked="false" role="menuitemradio" value={iframe}>
+			<svg className="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z" /></svg>
```

Comment:
Use the icon: https://github.com/sindresorhus/refined-github/blob/master/source/libs/icons.tsx#L3